metadata/workarounds.xml

```diff
@@ -68,5 +68,10 @@
 			<_long>Enable certain damage optimizations which are based on a surfaces' opaque regions. In some cases, this optimization might give unexpected results (i.e background app stops updating) even though this is fine according to Wayland's protocol.</_long>
 			<default>false</default>
 		</option>
+		<option name="force_frame_sync" type="bool">
+			<_short>Force frame synchronization.</_short>
+			<_long>This option can be used to workaround driver bugs that cause rendering artifacts, though can cause more resource usage. Leave disabled if unsure.</_long>
```

Comment:
I would add a comment saying that this helps with nvidia cards (because I expect that this will be the most common use-case for this option).

Reply:
LGTM otherwise.